src/account/PluginManagerInternals.sol

```diff
@@ -138,32 +133,21 @@ abstract contract PluginManagerInternals is IPluginManager {
         );
     }
 
-    function _installPlugin(address plugin, bytes32 manifestHash, bytes memory pluginInstallData) internal {
+    function _installPlugin(address plugin, PluginManifest calldata manifest, bytes memory pluginInstallData)
```

Comment:
It feels like we're converging on installing components rather than entire modules (prev: plugins), maybe we can gain some efficiency by directly installing those individual components (kind of like how we did with the validations).

One area of efficiency gain is in how we're obliged to pass an entire, even if pretty empty, manifest to install even a single exec hook, idk how legit this is, but it might be worth exploring.

Reply:
Gotcha, I don't think we need to change anything here for now, but it's probably worth keeping in mind for future updates, maybe we can simplify some installation paths.